core/src/main/scala/kafka/server/AutoTopicCreationManager.scala

```diff
@@ -159,8 +192,8 @@ class DefaultAutoTopicCreationManager(
   }
 
   private def sendCreateTopicRequest(
-    creatableTopics: Map[String, CreatableTopic],
-    metadataRequestContext: Option[RequestContext]
+                                      creatableTopics: Map[String, CreatableTopic],
+                                      requestContext: Option[RequestContext]
```

Comment:
nit: This indentation seems quite off compared to the other methods in this class.

Reply:
Done